components/src/Modal/Modal.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  BackDrop,
+  ModalLayout,
+  CloseIcon,
+  ModalTitle,
+  ModalContents,
+  ModalButton,
+  ModalButtonContainer,
+} from './Modal.styled';
+import { useContext, useEffect } from 'react';
+import { createContext } from 'react';
+import { createPortal } from 'react-dom';
+
+type ModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  children: React.ReactNode;
+  position?: 'center' | 'bottom';
+};
```

Comment:
각각의 props 에 대한 설명이 스토리북 문서상에 등장할 수 있도록 주석을 달아주시면 좋을 것 같습니다.

```suggestion
type ModalProps = {
  /** 모달 열림/닫힘 상태 */
  isOpen: boolean;
  /** 모달 닫기 함수 */
  onClose: () => void;
  /** 모달 내용 */
  children: React.ReactNode;
  /** 모달 위치 */
  position?: "center" | "bottom";
};
```

Reply:
너무 좋은 것 같아요. 이렇게 하면 정말 스토리북으로 "문서화"한다는게 와닿는 것 같아요.
감사합니다 🙇‍♀️

[a5e305d](https://github.com/woowacourse/react-modules/pull/101/commits/a5e305dd3a42f58194c8e3f5733b6467d0761729)